lib/experimental/fonts/font-library/class-wp-font-collection.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Font Collection class.
+ *
+ * This file contains the Font Collection class definition.
+ *
+ * @package    WordPress
+ * @subpackage Font Collection
```

Comment:
```suggestion
 * @subpackage Font Library
```

It's part of the Font Library.